python/cudf/cudf/tests/test_index.py

```diff
@@ -2470,3 +2470,29 @@ def test_index_type_methods(data, func):
         assert_eq(False, actual)
     else:
         assert_eq(expected, actual)
+
+
+@pytest.mark.parametrize(
+    "resolution", ["D", "H", "T", "min", "S", "L", "ms", "U", "us", "N"]
+)
+def test_index_datetime_ceil(resolution):
```

Comment:
Are there any existing tests for `Series.ceil` that we should copy/migrate over to here?

Reply:
Same for floor?

Reply:
```python
def test_series_ceil():
    arr = np.random.random(100) * 100
    sr = Series(arr)
    np.testing.assert_equal(sr.ceil().to_numpy(), np.ceil(arr))


def test_series_floor():
    arr = np.random.random(100) * 100
    sr = Series(arr)
    np.testing.assert_equal(sr.floor().to_numpy(), np.floor(arr))
```

these two tests are there in `test_unaops.py`, do we want to modify them?

Reply:
I think we're good then. No need to modify/copy these.